.github/workflows/docs.yml

```diff
@@ -0,0 +1,31 @@
+name: Deploy docs to GitHub pages
+
+on:
+  pull_request:
```

Comment:
i would only run it in branch master and this branch (for testing)

this would also rewrite docs in case we merge pr to some other branch